src/renderer/shared/core/types/notification.ts

```diff
@@ -0,0 +1,50 @@
+import { ProxyType, WalletType } from '@shared/core';
+import type { ID, AccountId, CallHash, ChainId, Timepoint } from '@shared/core';
+
+export const enum NotificationType {
+  MULTISIG_INVITE = 'MultisigAccountInvitedNotification',
+  MULTISIG_CREATED = 'MultisigCreatedNotification',
+  MULTISIG_APPROVED = 'MultisigApprovedNotification',
+  MULTISIG_EXECUTED = 'MultisigExecutedNotification',
+  MULTISIG_CANCELLED = 'MultisigCancelledNotification',
+
+  PROXY_CREATED = 'ProxyCreatedNotification',
+  PROXY_REMOVED = 'ProxyRemovedNotification',
+}
+
+type BaseNotification = {
+  id: ID;
+  read: boolean;
+  dateCreated: number;
+  type: NotificationType;
+};
+
+type MultisigBaseNotification = BaseNotification & {
+  multisigAccountId: AccountId;
+  originatorAccountId: AccountId;
+  smpRoomId: string;
+};
+
+export type MultisigInvite = MultisigBaseNotification & {
+  signatories: AccountId[];
+  threshold: number;
+  multisigAccountName: string;
+};
+
+export type MultisigOperation = MultisigBaseNotification & {
+  callHash: CallHash;
+  callTimepoint: Timepoint;
+  chainId: ChainId;
+};
+
+export type ProxyAction = BaseNotification & {
+  proxyAccountId: AccountId;
+  proxiedAccountId: AccountId;
+  proxyType: ProxyType;
+  chainId: ChainId;
+  proxyWalletName: string;
```

Comment:
Need to set this data in notification, otherwise we are likely to lose data when trying to map data from  `wallets/accounts` in UI